model/src/main/proto/event_logger.proto

```diff
@@ -104,6 +104,8 @@ message EventLog {
     OPEN_CONCEPT_CARD = 9;
     // Opening of revision card.
     OPEN_REVISION_CARD = 10;
+    // Opening of profile activity.
+    OPEN_PROFILE = 11;
```

Comment:
OPEN_PROFILE_CHOOSER maybe? This is not getting invoked when you open a profile but more when you are on the profile chooser page right?

Reply:
Done.